### PR TITLE
Slime Ore

### PR DESCRIPTION
**Pull request by Azael-Snively**
This it is Slime Ore

### CODE REVIEW

**Review by IntegratedQuantum — COMMENTED** [review]
First of all thank you for considering to contribute to Cubyz.

Apart from minor (mostly syntactical) issues, your pull request misses just one important thing:
### What is slime ore?
You probably have some great ideas in the back of your head, but if you don't write them down, we are not going to understand, why we should add slime ore into the game.

And now about those syntactical issues:
First of all: The file names.
The internal convention in Cubyz is to use lower-case words separated by "_" instead of " ".
So the correct file names would be `slime_ore` and `slime_ore.png`

File: cubyz-client/addons/cubyz/blocks/Slime  Ore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class ore
+
+hardeness 100
+veins 2
+size 1.4
+height 16
+
+drop cubyz:SlimeOre
```
[the remark this answers is not in the document]
Comment (Azael-Snively) [reply cubyz-client/addons/cubyz/blocks/Slime  Ore:8]: slime ore is the picture

File: cubyz-client/addons/cubyz/blocks/Slime  Ore [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class ore
+
+hardeness 100
```
[the remark this answers is not in the document]
Comment (Azael-Snively) [reply cubyz-client/addons/cubyz/blocks/Slime  Ore:3]: thanks i will change it

[the remark this answers is not in the document]
Comment (Azael-Snively) [reply cubyz-client/addons/cubyz/blocks/Slime  Ore:3]: thanks i will change it

**Comment by Azael-Snively** [comment]
Slime Ore  he will serve for make armor in slime it just like an ore

**Comment by Azael-Snively** [comment]
i fix the problem

**Comment by IntegratedQuantum** [comment]
Cubyz features a system that can automatically generate ores with corresponding tools (and armor, once we add it).
To compete with these randomly generated ores, your slime ore needs to offer more than that.

If you have no such things to offer, but still think that your slime ores needs to be added to the game, I suggest you write an [Addon](https://github.com/PixelGuys/Cubyz/wiki/Addons) instead of contributing to the base game.

**Comment by Azael-Snively** [comment]
Can You tell me what should i do for make a better pull request please

**Comment by IntegratedQuantum** [comment]
Your pull request is (mostly) fine.

The problem is the idea behind your pull request.

If you want your idea to get into the base game, you need to offer the player something useful or unique.

Slime ore as you present it is neither unique(there are lots of ores in the game already) nor useful(it can only be used for probably weak armor).

**Comment by Azael-Snively** [comment]
can you give me an example of what should i say to be convincing

**Comment by IntegratedQuantum** [comment]
If I knew what you should say, I would have said it already.
But I can try to give you examples of how other blocks/items are unique or useful:
#### Coal
- can be used as efficient fuel source(not implemented yet)
- can be used to craft torches(currently our only artificial light source)
#### Wood
- can be used for the workbench which is needed to make tools and progress.
- tools have a special property: `Regrowth` which slowly regrows a tool.
- It is good for decoration.
#### Stone
- It's unique because it is a very common decorative block
- it is useful because it makes the most efficient tools without having to mine for a long time.
#### Apples
- They are unique and useful because they are the only accessible food source(right now).
#### Cactus, Snow, Ice, Gravel, Mossy Cobblestone, Grass
- Adds variety to biomes
- Can be used for decoration
#### randomly generated ores
- Add variety to the underground.
- Can be used to make possibly useful tools/armor(partly implemented).
- Can be used for colored torches(planned).
- Add new things to discover when a new world is created.

**Comment by Azael-Snively** [comment]
Ok i see

**Comment by Azael-Snively** [comment]
Slime Ore it will be first for add variety to the underground it will be for make armor and weapon precisely the hammer(i will add it)
and the wand to it should be burn in a furnace for get slime ball

**Comment by Azael-Snively** [comment]
it was convincing

**Comment by Azael-Snively** [comment]
can you give me a link for discord server please

**Comment by IntegratedQuantum** [comment]
https://discord.gg/XtqCRRG
(The same link is also at the bottom of the Readme.)

**Comment by Azael-Snively** [comment]
so did my pull request was accepted

**Pull request closed without merging**